src/main/java/com/github/rodionovsasha/commentservice/services/CommentService.java

```diff
@@ -5,5 +5,7 @@
 public interface CommentService {
     Comment add(String content, long topicId, long userId);
 
-    void update(long commentId, long userId, String content);
+    void updateContent(long commentId, long userId, String content);
```

Comment:
Let's have by-the-way refactoring in a separate PR in the future - harder to read and to review